core/app/helpers/mno_enterprise/image_helper.rb

```diff
@@ -0,0 +1,36 @@
+module MnoEnterprise
+  module ImageHelper
+    
```

Comment:
add `_path` to all the method so that it's crital clear what they are returning.

- main_logo_white_bg_path
- main_logo_path